bindings/node/lib/clientEncryption.js

```diff
@@ -549,6 +558,65 @@ module.exports = function (modules) {
       return value;
     }
 
+    /**
+     * A convenience method for creating an encrypted collection.
+     * This method will create data keys for any encryptedFields that do not have a `keyId` defined
+     * and then create a new collection with the full set of encryptedFields.
+     *
+     * @template {TSchema} - Schema for the collection being created
+     * @param {Db} db - A database to create the collection in
+     * @param {string} name - The name of the collection to be created
+     * @param {object} options - Options for createDataKey and for createCollection.
```

Comment:
```suggestion
     * @param {object} options - Options for createDataKey and for createCollection
```

let's be consistent with either always ending with '.'s or never ending with them.